libwallet/src/slatepack/address.rs

```diff
@@ -43,8 +44,12 @@ pub struct SlatepackAddress {
 impl SlatepackAddress {
 	/// new with default hrp
 	pub fn new(pub_key: &edDalekPublicKey) -> Self {
+		let hrp = match global::get_chain_type() {
+			global::ChainTypes::Mainnet => "grin",
+			_ => "tgrin",
+		};
```

Comment:
Does it make sense to have something else for AutomatedTesting/UserTesting (Regnet?) ?
Something like:
```suggestion
		let hrp = match global::get_chain_type() {
			global::ChainTypes::Mainnet => "grin",
			global::ChainTypes::Floonet => "tgrin",
			// UserTesting or AutomatedTesting
			_ => "rgrin",
		};
```

Reply:
Possible, but my logic was that if someone's doing user testing or automated testing, it's in a local environment and they know what they're testing anyhow. If there's a call for it we can add another type in later.